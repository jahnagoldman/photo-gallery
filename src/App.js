// import logo from './logo.svg';
import './App.css';
import PhotoGallery from './PhotoGallery';
// import ocelot from './images/ocelot.jpg'
// import sunflower from './images/sunflower.jpg'

function App() {
  const images = [
  {
    'url': 'https://www.elmwoodparkzoo.org/wp-content/uploads/2018/05/1920x1080-Ocelotedit.jpg',
    'caption': 'picture 1'
  },
  {
    'url': 'https://www.jocooks.com/wp-content/uploads/2020/10/club-sandwich-1-9.jpg',
    'caption': 'picture 2'
  }
]
  return (
    <div className="App">

      <PhotoGallery images={images} />

    </div>
  );
}

export default App;
