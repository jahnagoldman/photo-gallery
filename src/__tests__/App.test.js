import App from '../App';
import React from 'react';
import renderer from 'react-test-renderer';

test('renders App component', () => {
  const component = renderer.create(<App/>);
  let app = component.toJSON();
  expect(app).toMatchSnapshot();
});