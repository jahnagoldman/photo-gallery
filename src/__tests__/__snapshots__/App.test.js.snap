// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<div>
  <div
    onMouseDown={[Function]}
  >
    <div
      className="PhotoContainer"
    >
      <img
        alt="new"
        onClick={[Function]}
        src="https://www.elmwoodparkzoo.org/wp-content/uploads/2018/05/1920x1080-Ocelotedit.jpg"
      />
      <div
        className="PhotoCaption"
      >
        Ocelot
      </div>
    </div>
  </div>
</div>
`;
